cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/sql/PostgreSqlStructuredTable.java

```diff
@@ -542,6 +542,13 @@ public void deleteAll(Range keyRange) throws InvalidFieldException, IOException
     }
   }
 
+  @Override
+  public void updateAll(Range keyRange, Collection<Field<?>> fields) throws InvalidFieldException, IOException {
+    LOG.trace("Table {}: Update fields {} in range {}", tableSchema.getTableId(), fields, keyRange);
+    fieldValidator.validateScanRange(keyRange);
```

Comment:
Same comment from the Spanner implementation, pls specify the `keyRange` needs to be primary keys in order and make sure that the passed fields are all valid non-key fields